sdk/elasticsan/Azure.ResourceManager.ElasticSan/src/autorest.md

```diff
@@ -36,12 +36,14 @@ rename-rules:
   SSO: Sso
   URI: Uri
   MBps: Mbps
+  LRS: Lrs
+  Volume: ElasticSanVolume
 directive:
   - from: elasticsan.json
-    where: "$.definitions"
+    where: '$.definitions'
     transform: >
-      $.Sku.properties.name["x-ms-enum"]["name"] = "ElasticSanSkuName";
-      $.Sku.properties.tier["x-ms-enum"]["name"] = "ElasticSanTier";
-      $.VirtualNetworkRule.properties.state["x-ms-enum"]["name"] = "VirtualNetworkRuleState";
+      $.Sku.properties.name['x-ms-enum']['name'] = 'ElasticSanSkuName';
+      $.Sku.properties.tier['x-ms-enum']['name'] = 'ElasticSanTier';
+      $.VirtualNetworkRule.properties.state['x-ms-enum']['name'] = 'VirtualNetworkRuleState';
```

Comment:
It seems we don't need these configurations because the initial generated code has those changes accordingly.

Reply:
no, that's only for model name not property nam